BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\DependencyInjection\Compiler;
+
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\AdministrationPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\ContentTypeForContentPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\GeneralNodesPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\GSTreeTemplatePanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TopMenuPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TreeNodesPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TreeTemplatePanelStrategy;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Class RoleCompilerPass
+ */
+class RoleCompilerPass implements CompilerPassInterface
+{
+    /**
+     * You can modify the container here before it is dumped to PHP code.
+     *
+     * @param ContainerBuilder $container
+     *
+     * @api
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $roles = array(
```

Comment:
I'm not sure it's a good idea to allow an iintegrator to remove basic roles. Add new one ok, but remove, I'm not sure